src/reducers/CreateUserReducer.ts

```diff
@@ -1,26 +1,10 @@
 import { Action } from '../types/ReduxTypes'
 
 export interface CreateState {
-  readonly confirmPassword: string | null
-  readonly confirmPasswordErrorMessage: string | null
   readonly createErrorMessage: string | null
-  readonly createPasswordErrorMessage: string | null
-  readonly password: string | null
-  readonly pin: string
-  readonly pinErrorMessage: string | null
-  readonly username: string | null
-  readonly usernameErrorMessage: string | null
 }
 
 const initialState: CreateState = {
-  username: null,
-  password: null,
-  confirmPassword: null,
-  pin: '',
-  createPasswordErrorMessage: null,
-  confirmPasswordErrorMessage: null,
-  pinErrorMessage: null,
-  usernameErrorMessage: null,
   createErrorMessage: null
 }
```

Comment:
You could delete this whole reducer if you got rid of `createErrorMessage`.

It is used inside `NewAccountTosScene`, and doesn't even need to be stored as state. Instead, when `handleNextPress` fails, just call `Alert.alert` to show the error directly, instead of doing the redux loop-the-loop.